src/expire.c

```diff
@@ -489,6 +489,47 @@ int checkAlreadyExpired(long long when) {
     return (when <= mstime() && !server.loading && !server.masterhost);
 }
 
+#define FLAG_NO_FLAGS 0
+#define FLAG_NX (1<<0)
+#define FLAG_XX (1<<1)
+#define FLAG_GT (1<<2)
+#define FLAG_LT (1<<3)
```

Comment:
Would it be better to change it to `EXPIRE_*`? Like `ZADD_NX`.

Reply:
I think these options are not frequently used like `SETNX` so use them as options can be sufficient.

Reply:
i think you may be missing the point, or maybe it's me..
i think the ask was for a better prefix to the defines rather than a generic "FLAG".

Reply:
Sorry, what I mean is prefix.

Reply:
I see :joy: